go/teams/loader.go

```diff
@@ -804,9 +798,6 @@ func (l *TeamLoader) load2InnerLockedRetry(ctx context.Context, arg load2ArgT) (
 	// If we did get an update from the server (hiddenResp != nil) are not a
 	// restricted bot AND this is not a recursive load (arg.readSubteamID == nil),
 	// then the server should have given us hidden chain data.
-	if hiddenResp != nil && hiddenResp.RespType == libkb.MerkleHiddenResponseTypeNONE && !role.IsRestrictedBot() && arg.readSubteamID == nil {
```

Comment:
Why are we taking this out? 
This is about whether the hidden chain is there or not, not about whether links were committed to the blind tree.